.travis.yml

```diff
@@ -57,10 +57,15 @@ before_script:
   - $RUN pip install --upgrade numpy
 
   - $RUN pip install -U pyarrow==${PYARROW_VERSION} tensorflow==${TF_VERSION}
+
+  # TODO(yevgeni): need to update docker build image at some stage
+  - $RUN pip install -U torchvision==0.2.1 torch==1.5.1 Pillow==7.1.2
```

Comment:
Can we do this [in the Dockerfile](https://github.com/uber/petastorm/blob/master/docker/Dockerfile#L34)? Should be a really quick one-line change, I think. Possibly saves us a nasty headache later on if trying to debug package issues.

Reply:
Done. Going to land the PR.